pkgs/development/python-modules/pymupdf/default.nix

```diff
@@ -1,28 +1,53 @@
-{ lib, buildPythonPackage, fetchPypi, mupdf, swig }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchpatch
+, mupdf
+, swig
+, freetype
+, harfbuzz
+, openjpeg
+, jbig2dec
+, libjpeg_turbo
+, gumbo
+}:
+
 buildPythonPackage rec {
   pname = "pymupdf";
-  version = "1.18.0";
+  version = "1.18.16";
 
   src = fetchPypi {
     pname = "PyMuPDF";
     inherit version;
-    sha256 = "64ce58f92d9edd2631e447175fa13b4024ac3b6dce6e718e0b003c41de6f7952";
+    sha256 = "b21e39098fbbe0fdf269fdb2d1dd25a3847bbf22785ee8903d3a5637c2d0b9d7";
   };
 
-  patchPhase = ''
+  patchFlags = [ "--binary" "--ignore-whitespace" ];
```

Comment:
Why are those flags necessary especially binary?

Reply:
I local test found the `setup.py` is DOS type, but the patch file is `UTF-8`, so the `patch` command will failed.